templates/editor.recurexception-question.php

```diff
@@ -0,0 +1,18 @@
+<div>
+	<form class="events" ng-submit="">
+		<fieldset class="events--fieldset">
+			<h2><?php p($l->t('Do you want to change only this or this and all future occurrences?')); ?></h2>
```

Comment:
What about 'Do you want to change this only or this and all future occurrences'?
Think the 'this or this', is a little confusing when reading it fast.